src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
```

Comment:
Do we want this in a lib ?

Reply:
This came with CRA and I didn't bother to delete it. Rest assured, this doesn't go into the published lib code. Only the contents of the `lib` folder are processed by the `npm run build` command.

Reply:
Yes I know it's the CRA that generated it, but do we need it ? Or can we remove it

Reply:
To be honest, I don't really know what this file is for. Will delete it, run `npm start` to check for broken imports and commit if everything's ok.

Reply:
It repport the web vitals for web apps, it's usefull for apps but not so much for libraries, see more here : https://web.dev/vitals/